src/arrow1/metadata.rs

```diff
@@ -0,0 +1,229 @@
+use wasm_bindgen::prelude::*;
+
+/// Global Parquet metadata.
+#[derive(Debug, Clone)]
+#[wasm_bindgen]
+pub struct ParquetMetaData(parquet::file::metadata::ParquetMetaData);
+
+#[wasm_bindgen]
+impl ParquetMetaData {
+    /// Returns file metadata as reference.
+    #[wasm_bindgen]
```

Comment:
I know I didn't do this originally in my PR, but either in this PR (or I'm happy to make a follow up PR myself) we should do a pass to ensure everything is camel-cased for JS